src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -1,12 +1,16 @@
 package core.basesyntax;
 
 public class BinaryString {
-
-    /**
-     * Напишите метод toBinaryString(), который принимает на вход целое число value,
-     * а возвращает String с представлением этого числа в двоичном виде.
-     */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+
+        StringBuffer result = new StringBuffer();
```

Comment:
read [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html) attentively

Reply:
Changed to StringBuilder. Will ask you tomorrow, why not StringBuffer or String.format() for example :)

Reply:
https://www.javatpoint.com/difference-between-stringbuffer-and-stringbuilder